js/gated-content/src/store/modules/auth/daxkosso.js

```diff
@@ -0,0 +1,43 @@
+import client from '@/client';
+
+export default {
+  state: {
+    config: null,
+  },
+  actions: {
+    async daxkossoAuthorize(context) {
+      if (!context.getters.isLoggedIn) {
+        const urlParams = new URLSearchParams(window.location.search);
+        const state = urlParams.get('state');
+        const code = urlParams.get('code');
+
+        if ((state !== null) && (code !== null)) {
+          const apiCheckEndPoint = `/openy-gc-auth-daxko-sso/back-redirect?state=${state}&code=${code}`;
```

Comment:
Maybe make sense to move `/openy-gc-auth-daxko-sso/back-redirect`to the plugin config. In this case, developers can create custom routes with additional logic and override this on backend.